### ISSUE TITLE
FinalBurn Neo not loading Neo Geo settings correctly

### ISSUE THREAD

**Issue by ribolovac510** [Turro75/MyMinUI#31]
Hi,

I've tried to set the Universe Bios v4.0 on Metal Slug and other Neo Geo games, either as a game or console setting, but the settings would never get loaded during the initial launch from the menu. Only after resetting the game with a hotkey the Universe Bios would load.

After quitting the game and launching it again, the default MVS bios is loaded instead of the set bios. I've tried changing both the UNIBIOS and the dipswitch settings but the behavior is always the same.

Here's a log file of a single run: [NG.txt](https://github.com/user-attachments/files/18138288/NG.txt)

**Comment by Turro75** [comment]
Seems a core or wrong bios issue, can You share also the cfg used?

**Comment by ribolovac510** [comment]
Yes, OG RG35XX. I've also tried setting other bioses and moving the neogeo.zip file to the fbneo folder inside the regular FBN/NG bios folder, just in case there's a detection issue, but it always runs the default "MVS Asia/Europe ver. 6 (1 slot)" bios first no matter which bios is set in the settings. Here's the config file: [minarch.zip](https://github.com/user-attachments/files/18148343/minarch.zip)

**Comment by Turro75** [comment]
I never seen this as a an issue, which is the plus in setting different bios?
Have You tried launching the game with retroarch? does it work as expected?

**Comment by ribolovac510** [comment]
Universe Bios offers you to switch between MVS and AES hardware modes and change country regions, it exposes all hardware and software dipswitches and also allows you to bypass HW checks at rom boot. If you like messing around it also offers a built-in cheat database and jukebox player.

On RetroArch it works as expected, it loads the correct bios you've set during launch. It has some other problems though but it's not related to this issue.

**Comment by Turro75** [comment]
Fixed the issue, minarch didn't load the core settings at start, many cores didn't care but arcades (fbneo and mame) were affected. Now it's fine.
